internal/invocation/stripe_executor_test.go

```diff
@@ -0,0 +1,158 @@
+package invocation
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/goleak"
+)
+
+type orderChecker struct {
+	sync.Mutex
+	previousCallArg int
+}
+
+func (oc *orderChecker) call(arg int) {
+	oc.Lock()
+	defer oc.Unlock()
+	if oc.previousCallArg+1 != arg {
+		panic("order of the tasks is not preserved")
+	}
+	oc.previousCallArg = arg
+}
+
+func Test_defaultExecFn(t *testing.T) {
+	var oc orderChecker
+	tasks := make(chan func(), 10_000)
+	for i := 1; i < 4; i++ {
+		tmp := i
+		tasks <- func() {
+			oc.call(tmp)
+		}
+	}
+	quit := make(chan struct{})
+	var wg sync.WaitGroup
+	wg.Add(1)
+	go defaultExecFn(tasks, quit, &wg)
+	assert.Eventually(t, func() bool {
+		return oc.previousCallArg == 3
+	}, time.Second, time.Millisecond*200, "execute function could not finish the tasks")
```

Comment:
How about `200*time.Millisecond` instead of `time.Millisecond*200`?

Reply:
Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/696/commits/db4dcd61d1d5f15c2887eabffc894e8c33884651